src/components/SafeRedirect.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { InjectAppServices } from '../services/pure-di';
+import RedirectToExternalUrl from './RedirectToExternalUrl';
+import RedirectToLogin from './RedirectToLogin';
+
+/**
+ * @param { Object } props
+ * @param { string } props.to
+ * @param { import('../services/pure-di').AppServices } props.dependencies
+ */
+function SafeRedirect({ to, whitelist, dependencies }) {
+  const loginWhitelist =
+    [
+      'https://academy.fromdoppler.com/',
+      'http://prod.doppleracademy.com/',
+      'https://doppleracademy.com/',
+    ] || whitelist;
+
+  const isPartialUrl = (url) => {
+    return RegExp(/^\/[\/\w*]*[\w\?\&\=]*$/).test(url); // eslint-disable-line
+  };
+
+  // whitelist = whitelist || loginWhitelist;
+
+  const whitelisted = (url, whitelist) => {
+    return whitelist && whitelist.find((element) => url.indexOf(url) !== -1);
+  };
+
+  if (isPartialUrl(to)) {
+    return <RedirectToExternalUrl to={dependencies.appConfiguration.dopplerLegacyUrl + to} />;
+  } else if (whitelisted(to, loginWhitelist)) {
```

Comment:
I think that making this function parametrizable is not necessary, I mean it has a very clear responsibility, it is not necessary making this reusable.

Reply:
I still think it add readability, it's quicker to understand.

Reply:
In that case, we should disable the rule globally because we do not want to take it into account.

Reply:
Pedon, confundí el comentario, luego lo hablamos.